### ISSUE TITLE
Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by shobhit-v** [shobhit-v/demo#117]
## Issue Description
**Issue Type:** Accessibility
**Impact:** Serious
**Checkpoint Group:** 1.1 Text Alternatives
**Checkpoint:** Alternative Text (Informative Images) (1.1.1.b)
**Method:** Automated
**Issue Summary:** Elements must have sufficient color contrast
**Issue Description:** Elements must have sufficient color contrast

**Source Code:** <p>Gefälscht CompuTech</p>## Test Case Description
**Test Case Name:** Test
**WCAG Standards:** wcag2a, wcag2aa, wcag21a, wcag21aa
## Test Run Description
**Test Run Name:** Test_desktop_09-03-20-10:53
**Digital Asset Type:** Desktop Web
**Release:** Desktop Web
**Environment:** Desktop Web
**Platform:** Desktop Web
**Assistive technology:** Desktop Web
## Test Unit Description
**Unit Type:** Page
**Test Unit Name:** Page
**Tested On URL:** http://abcdcomputech.dequecloud.com/
## Expected behavior
 RULE :
The alternative text for informative images MUST be accurately conveying the purpose of the image, and the author&#039;s intent in a way that is useful to those who cannot see the image.

HOW TO FIX:
Fix this issue by ensuring that the alternative text for the informative image describes its intent, purpose, or meaning.

&lt;img src=&quot;sunset.jpg&quot; alt=&quot;The sun setting over the Pacific Ocean, with a silhouette of a flying seagull in the foreground&quot;&gt;

REFERENCE:
Deque University: https://dequeuniversity.com/class/images2/alt-text/informative
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

BACKGROUND:
People who are blind cannot see images on a page. In order to give people who cannot see an image access to the information conveyed by the image, it must have a text alternative. The text alternative must describe the information or function represented by the image. Screen readers can then use the alternative text to convey that information to the screen reader user.

For informative images - images that convey information - the alternative text must communicate the intent, purpose, or meaning of the image in a way that serves as a true alternative for the image.
## More Info
 https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=axeAPI
## Issue Comments
 shobhit-admin: test comment 1,shobhit-admin: test comment 2
